src/components/heading/heading.stories.mdx

```diff
@@ -42,6 +42,18 @@ import Heading from "carbon-react/lib/components/heading";
   </Story>
 </Canvas>
 
+### Heading with headingType
+
+The `headingType` prop changes the HTML heading element of the `title` string which is displayed within the component. Each `string` 
+represents a HTML Section Heading element e.g `h1` is equal to `<h1>`, allowing up to six sections of headings from most to least important.
+
+<Canvas>
+  <Story name="with headingType">
+    <Heading headingType="h2" title="This is a h2 Title" />
```

Comment:
a really minor suggestion but it might be clearer what's going on if you change the example from h2 to something more obviously contrasting with h1, such as an h4 or h5.